tests/test_create.py

```diff
@@ -1351,42 +1319,6 @@ def test_install_update_deps_only_deps_flags(
         assert package_is_installed(prefix, "dependency=2.0")
 
 
-@pytest.mark.skipif(on_win, reason="nomkl not present on windows")
-def test_install_features(
```

Comment:
I assume here too? New name under `test_features`?

Reply:
Correct, moved it and expanded on it